### PR TITLE
[webpack-manifest-plugin] Added typings

### PR DESCRIPTION
**Pull request by r3nya**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If adding a new definition:
- [x] The package does not provide its own types, and you can not add them.
- [x] If this is for an NPM package, match the name. If not, do not conflict with the name of an NPM package.
- [x] Create it with `dts-gen --dt`, not by basing it on an existing project.
- [x] `tslint.json` should be present, and `tsconfig.json` should have `noImplicitAny`, `noImplicitThis`, `strictNullChecks`, and `strictFunctionTypes` set to `true`.

### CODE REVIEW

**Comment by r3nya** [comment]
Issue https://github.com/DefinitelyTyped/DefinitelyTyped/issues/21497.

File: types/webpack-manifest-plugin/webpack-manifest-plugin-tests.ts [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { Configuration } from 'webpack';
+import * as WebpackManifestPlugin from 'webpack-manifest-plugin';
```
Comment (sandersn) [inline types/webpack-manifest-plugin/webpack-manifest-plugin-tests.ts]: Same comments as your other PR

**Review by sandersn — CHANGES_REQUESTED** [review]

**Comment by r3nya** [comment]
@sandersn I've updated typings and test file. Please check it. Thanks! 😉

**Review by sandersn — APPROVED** [review]

**Comment by r3nya** [comment]
@sandersn thanks man! ❤️

**Pull request merged**